lib/membrane/rtp/serializer.ex

```diff
@@ -85,10 +85,11 @@ defmodule Membrane.RTP.Serializer do
       extension: Map.get(rtp_metadata, :extension)
     }
 
-    packet = %RTP.Packet{header: header, payload: payload}
+    packet = %RTP.Packet{header: header, payload: buffer.payload}
     payload = RTP.Packet.serialize(packet, align_to: state.alignment)
 
     buffer = %Buffer{
+      pts: Buffer.get_dts_or_pts(buffer),
```

Comment:
Here we potentially set dts as pts, and set dts to nil... why?

Reply:
Yes, you're right, that's probably messed up, I need to rethink that a bit.